.github/pull_request_template.md

```diff
@@ -4,6 +4,8 @@
 
 - [ ] I have read through the [README](https://github.com/wfxr/minimap.vim/blob/master/README.md) (especially F.A.Q section)
 - [ ] I have searched through the existing issues or pull requests
+- [ ] I have verified all existing unit tests pass (See the README on how to run)
```

Comment:
This check can be removed once we automate the testing.

Reply:
Adding the following steps into the ci workflow seems to be a feasible solution?

```sh
mkdir -p ~/.vim/autoload
curl -fLo ~/.vim/autoload/plug.vim https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
# generate a minimal `vimrc` which loads `vim-testify` and `minimap.vim`
vim +PlugInstall +qall
vim +TestifySuite
```